Source/Resources/Content/MSGraph/DeviceManagement/DeviceConfigurations/OS - Preview Windows 10 Update Ring.json

```diff
@@ -19,7 +19,7 @@
   "deadlineForFeatureUpdatesInDays": 5,
   "deadlineForQualityUpdatesInDays": 3,
   "deadlineGracePeriodInDays": 1,
-  "description": "Preview Windows Update configuration for devices.",
+  "description": "What is it: Creates a Windows Update configuration for Intune managed devices that gets updates after Insiders but before devices with default policy. \n Simeon Notes: A Windows update ring is the best way to ensure Windows updates are compatible in your environment. The preview ring is meant to be applied to a subset of users that use a variety of programs. Simeon's baseline delays feature and quality updates to the preview group for 7 days after released by Microsoft. Once the deferral period has expired for the device, users have 3 days to restart (if required). Also, unattended updates are only be applied outside the working hours of 5am to 10pm. Applies to users in the Azure AD group 'BUILTIN - Corporate Devices – Preview'. \n Learn More: https://techcommunity.microsoft.com/t5/windows-it-pro-blog/tactical-considerations-for-creating-windows-deployment-rings/ba-p/746979",
```

Comment:
**What does this do?**
Creates a Windows Update configuration for those Intune-managed devices in the Preview ring that receive updates after Insiders but before devices with the default policy. The baseline delays feature and quality updates to the preview group for 7 days after released by Microsoft. Once the deferral period has expired for the device, users have 3 days to restart (if required). Unattended updates are only be applied outside the working hours of 5am to 10pm. Applies to users in the Azure AD group 'Baseline - Corporate Devices – Preview'. 

**Why should you use this?**
A Windows update ring is the best way to ensure Windows updates are compatible in your environment. 

**What is the end-user impact?**
Users in the Preview ring will receive updates 7 days after being released by Microsoft and after Insiders, but before devices with the default policy. 

**Learn more**
[Tactical considerations for creating Windows deployment rings](https://techcommunity.microsoft.com/t5/windows-it-pro-blog/tactical-considerations-for-creating-windows-deployment-rings/ba-p/746979)